### PR TITLE
create order

### PR DESCRIPTION
**Pull request by thucpv-1748**
https://edu-redmine.sun-asterisk.vn/issues/9714

### CODE REVIEW

File: app/Http/Controllers/Backend/DashboardController.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public $order;
+
+    /**
+     * DashboardController constructor.
+     * @param FilmInterface $film
+     * @param CategoryInterface $category
+     * @param UserInterface $user
+     * @param OrderInterface $order
+     */
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 39</br>Line exceeds 120 characters; contains 125 characters

File: app/Http/Controllers/Backend/DashboardController.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        $this->category = $category;
+        $this->film = $film;
+        $this->user = $user;
+        $this->order = $order;
+    }
+
+    /**
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 51</br>Opening brace should be on a new line

File: app/Http/Controllers/Backend/OrderController.php [hunk 3]
```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Repositories\Contracts\OrderInterface;
+use Illuminate\Support\Facades\Auth;
+use App\Repositories\Contracts\CouponInterface;
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 11</br>There must be one blank line after the last USE statement; 2 found;

File: app/Http/Controllers/Backend/OrderController.php [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+     * @param OrderInterface $model
+     * @param CouponInterface $coupon
+     * @param TimeShowInterface $timeShow
+     * @param FastFoodInterface $fastFood
+     */
+    public function __construct(
+        OrderInterface $model,
+        CouponInterface $coupon,
+        TimeShowInterface $timeShow,
+        FastFoodInterface $fastFood
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 50</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: app/Http/Controllers/Backend/OrderController.php [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+        $coupon  = $this->coupon->where('status', '=', '1')->get();
+
+        return view('layout.backend.orderform', compact('fastFood', 'timeShow', 'coupon'));
+    }
+
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function createOrder(Request $request)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 76</br>Expected 1 space(s) after TRY keyword; 0 found

File: app/Http/Controllers/Backend/OrderController.php [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    {
+        try{
+            $this->model->create($request->all());
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 79</br>No space found after comma in function call

File: app/Http/Controllers/Backend/OrderController.php [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 80</br>Expected 1 space after closing brace; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 80</br>Expected 1 space(s) after CATCH keyword; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 80</br>Expected 1 space(s) after closing parenthesis; found 0

File: app/Http/Controllers/Backend/OrderController.php [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+        }catch(\Exception $e){
+            // insert query
+            return redirect()->back()->with('error', $e->getMessage());
+        }
+    }
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function updateOrder(Request $request)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 91</br>Expected 1 space(s) after TRY keyword; 0 found

File: app/Http/Controllers/Backend/OrderController.php [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+    {
+        try{
+            $this->model->update($request->id, $request->all());
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 94</br>No space found after comma in function call

File: app/Http/Controllers/Backend/OrderController.php [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 95</br>Expected 1 space after closing brace; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 95</br>Expected 1 space(s) after CATCH keyword; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 95</br>Expected 1 space(s) after closing parenthesis; found 0

File: app/Http/Controllers/Backend/OrderController.php [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+            return redirect()->back()->with('error', $e->getMessage());
+        }
+    }
+
+
+    /**
+     * @param $id
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function ajaxGetCoupon($id)
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 108</br>Line indented incorrectly; expected at least 8 spaces, found 7

File: app/Http/Controllers/Backend/OrderController.php [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+    {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 109</br>Line indented incorrectly; expected 8 spaces, found 7

File: app/Http/Controllers/Backend/OrderController.php [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+       $coupon = $this->coupon->findOrFail($id);
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 110</br>Line indented incorrectly; expected at least 12 spaces, found 11

File: app/Http/Controllers/Backend/OrderController.php [hunk 14]
```diff
@@ -0,0 +1,160 @@
...
+       if ($coupon) {
+           $data = $coupon->toJson();
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 112</br>Line indented incorrectly; expected at least 12 spaces, found 11

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ShortArraySyntax</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 112</br>"array()" function used. Use short array syntax, "[]", instead.

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 112</br>Expected 1 space between comma and "'data'"; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceBeforeDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 112</br>Expected 1 space between double arrow and "'data'"; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 112</br>Expected 1 space between double arrow and "$data"; 0 found

File: app/Http/Controllers/Backend/OrderController.php [hunk 15]
```diff
@@ -0,0 +1,160 @@
...
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 113</br>Line indented incorrectly; expected 8 spaces, found 7

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 113</br>Expected 1 space after closing brace; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 113</br>Expected 1 space(s) after ELSE keyword; 0 found

File: app/Http/Controllers/Backend/OrderController.php [hunk 16]
```diff
@@ -0,0 +1,160 @@
...
+           return response()->json(array('success' => true,'data'=>$data));
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.Incorrect</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 114</br>Line indented incorrectly; expected at least 12 spaces, found 11

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ShortArraySyntax</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 114</br>"array()" function used. Use short array syntax, "[]", instead.

File: app/Http/Controllers/Backend/OrderController.php [hunk 17]
```diff
@@ -0,0 +1,160 @@
...
+       }else{
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 115</br>Line indented incorrectly; expected 8 spaces, found 7

File: app/Http/Controllers/Backend/OrderController.php [hunk 18]
```diff
@@ -0,0 +1,160 @@
...
+
+    /**
+     * @param $id
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function ajaxGetTimeShow($id)
+    {
+        $timeShow = $this->timeShow->findOrFail($id);
+        if ($timeShow && $timeShow->status == 1) {
+            $timeShow->room->seat;
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 129</br>Variable "time_show" is not in valid camel caps format

File: app/Http/Controllers/Backend/OrderController.php [hunk 19]
```diff
@@ -0,0 +1,160 @@
...
+            $timeShow->order;
+            $time_show = $timeShow->toJson();
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ShortArraySyntax</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 131</br>"array()" function used. Use short array syntax, "[]", instead.

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 131</br>Variable "time_show" is not in valid camel caps format

File: app/Http/Controllers/Backend/OrderController.php [hunk 20]
```diff
@@ -0,0 +1,160 @@
...
+
+            return response()->json(array('success' => true, 'data' => $time_show));
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.ShortArraySyntax</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 133</br>"array()" function used. Use short array syntax, "[]", instead.

File: app/Model/Order.php [hunk 21]
```diff
@@ -42,4 +53,28 @@ public function fastFood()
...
+     */
+    public function setSeatAttribute($seat)
+    {
+        $this->attributes['seat'] = is_array($seat) ? implode(',', $seat) : $seat;
+    }
+
+    /**
+     * @param $user_id
+     */
+    public function setUserId($user_id)
```
Comment (framgiaciv3) [inline app/Model/Order.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Model/Order.php</br>**Line**: 68</br>Variable "user_id" is not in valid camel caps format

File: app/Http/Controllers/Backend/DashboardController.php [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+    public $order;
+
+    /**
+     * DashboardController constructor.
+     * @param FilmInterface $film
+     * @param CategoryInterface $category
+     * @param UserInterface $user
+     * @param OrderInterface $order
+     */
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.OneParamPerLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 40</br>Multi-line function declarations must define one parameter per line

File: app/Http/Controllers/Backend/DashboardController.php [hunk 23]
```diff
@@ -0,0 +1,62 @@
...
+    public function __construct(
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.OneParamPerLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 41</br>Multi-line function declarations must define one parameter per line

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 41</br>The closing parenthesis of a multi-line function declaration must be on a new line

File: app/Http/Controllers/Backend/DashboardController.php [hunk 24]
```diff
@@ -0,0 +1,62 @@
...
+        FilmInterface $film, CategoryInterface $category,
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 42</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: app/Http/Controllers/Backend/OrderController.php [hunk 25]
```diff
@@ -0,0 +1,158 @@
...
+    /**
+     * OrderController constructor.
+     * @param OrderInterface $model
+     * @param CouponInterface $coupon
+     * @param TimeShowInterface $timeShow
+     * @param FastFoodInterface $fastFood
+     */
+    public function __construct(
+        OrderInterface $model,
+        CouponInterface $coupon,
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 47</br>The closing parenthesis of a multi-line function declaration must be on a new line

File: app/Http/Controllers/Backend/OrderController.php [hunk 26]
```diff
@@ -0,0 +1,158 @@
...
+        TimeShowInterface $timeShow,
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/OrderController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/OrderController.php</br>**Line**: 48</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: app/Http/Controllers/Backend/DashboardController.php (hunk 22 above)
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.OneParamPerLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 40</br>Multi-line function declarations must define one parameter per line

File: app/Http/Controllers/Backend/DashboardController.php (hunk 23 above)
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.OneParamPerLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 41</br>Multi-line function declarations must define one parameter per line

Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 41</br>The closing parenthesis of a multi-line function declaration must be on a new line

File: app/Http/Controllers/Backend/DashboardController.php (hunk 24 above)
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/DashboardController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/DashboardController.php</br>**Line**: 42</br>The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line

File: resources/js/backend/order.js [hunk 27]
```diff
@@ -0,0 +1,144 @@
...
+                                    var checked = (data_seat.indexOf(value) > -1) ? 'checked="checked"' : '' ;
+                                    var checked_all =  (seat_order_ids.indexOf(value) > -1) ?'seat_selected':'';
+                                    html += '<li><p>\n' + (j+1) +'</p>';
+                                    html += '<input type="checkbox" name="seat[]" value="' + value + '" hidden="hidden" class="seat_ids" ' + checked + '><div class="' + checked_all + '"></div> ' + '</li>\n'
+                                }
+                                html +=   '</ul>';
+                            }
+                            $('.seat').append(html);
+                            selectSeat();
+                        }else{
```
Comment (BunnyPi04) [inline resources/js/backend/order.js]: convention

File: resources/js/backend/order.js [hunk 28]
```diff
@@ -0,0 +1,144 @@
...
+                            var html = '<h2>' + Lang.get('Room') + '</h2>\n';
+                            for(var i = 0; i < seat.length; i++){
+                                html += '<ul>\n' +
+                                    '<p>'+ seat[i].row +'</p>\n';
+                                for (var j = 0; j < seat[i].col; j++)
+                                {
+                                    var value = i + '-' + j;
+                                    var checked = (data_seat.indexOf(value) > -1) ? 'checked="checked"' : '' ;
+                                    var checked_all =  (seat_order_ids.indexOf(value) > -1) ?'seat_selected':'';
+                                    html += '<li><p>\n' + (j+1) +'</p>';
```
Comment (BunnyPi04) [inline resources/js/backend/order.js]: convention

File: resources/js/backend/order.js [hunk 29]
```diff
@@ -0,0 +1,144 @@
...
+                        if(seat.length){
+                            var html = '<h2>' + Lang.get('Room') + '</h2>\n';
+                            for(var i = 0; i < seat.length; i++){
+                                html += '<ul>\n' +
+                                    '<p>'+ seat[i].row +'</p>\n';
+                                for (var j = 0; j < seat[i].col; j++)
+                                {
+                                    var value = i + '-' + j;
+                                    var checked = (data_seat.indexOf(value) > -1) ? 'checked="checked"' : '' ;
+                                    var checked_all =  (seat_order_ids.indexOf(value) > -1) ?'seat_selected':'';
```
Comment (BunnyPi04) [inline resources/js/backend/order.js]: convention

File: resources/js/backend/order.js [hunk 30]
```diff
@@ -0,0 +1,144 @@
...
+            final = total_price - sale_price;
+        } else if (type == 2 && total_price > 0) {
+            final = total_price - ((total_price * sale_price) / 100);
+        } else {
+            final = total_price;
+        }
+        final = (final < 0 || !final) ? 0 : final;
+        $('#final_price').val(final);
+    }
+});
```
Comment (BunnyPi04) [inline resources/js/backend/order.js]: thiếu dòng trắng cuối file

File: resources/views/layout/backend/dashboard.blade.php [hunk 31]
```diff
@@ -0,0 +1,91 @@
+@extends('layout.backend.admin')
+
+
```
Comment (BunnyPi04) [inline resources/views/layout/backend/dashboard.blade.php]: xóa dòng trắng dư

File: resources/views/layout/backend/dashboard.blade.php [hunk 32]
```diff
@@ -0,0 +1,91 @@
...
+                    </div>
+                </div>
+            </div><!--/.col-->
+        </div><!--/.row-->
+    </div>
+    <script type="text/javascript">
+        $(document).ready(function() {
+        });
+    </script>
+@endsection
```
Comment (BunnyPi04) [inline resources/views/layout/backend/dashboard.blade.php]: thiếu dòng trắng cuối file

File: resources/views/layout/backend/order.blade.php [hunk 33]
```diff
@@ -0,0 +1,79 @@
...
+                @if ($order)
+                    @foreach ($order as $key => $value)
+                        <tr>
+                            <td>{{ $value->id }}</td>
+                            <td>{{ $value->user->name }}</td>
+                            <td>{{ isset($value->coupon) ? $value->coupon->name : '' }}</td>
+                            <td>{{ $value->timeShow->time_show }}</td>
+                            <td>{{ $value->timeShow->film->title }}</td>
+                            <td>{{ $value->fast_food_ids }}</td>
+                            <td>{{ config('status.'.$value->status) }}</td>
```
Comment (BunnyPi04) [inline resources/views/layout/backend/order.blade.php]: convention

**Pull request merged**